kivy/input/providers/mouse.py

```diff
@@ -46,6 +46,19 @@
     if 'multitouch_sim' in touch.profile:
         touch.multitouch_sim = True
 
+.. versionchanged:: 2.1.0
+
+Provider dispatches hover events by listening to properties/events in
+:class:`~kivy.core.window.Window`. Dispatching can be disabled by setting
+`True` to :attr:`MouseMotionEventProvider.disable_hover` property or by adding
```

Comment:
```suggestion
:attr:`MouseMotionEventProvider.disable_hover` to ``True`` or by adding
```

Switching order, but also, you need to double the backticks for sphinx to display the text as code.

Reply:
It is meant as highlight and not as a code. Like in `ModalView` https://github.com/kivy/kivy/blob/master/kivy/uix/modalview.py#L59

Reply:
Or like in https://github.com/kivy/kivy/blob/master/kivy/animation.py#L74 . 

So keep \`True\` or use \`\`True\`\`?

Reply:
Oh, I see. Personally I would still use ` ``True`` ` because it's code (it's capitalized). If you just wanted to highlight it then I'd write `` `true` `` because that's more regular english. But whichever you choose is fine.

Reply:
In that case I would rather use ` ``True`` ` as that is the exact value.